app/src/main/java/pl/droidevs/books/addBook/AddBookActivity.java

```diff
@@ -0,0 +1,144 @@
+package pl.droidevs.books.addBook;
```

Comment:
zgodnie z https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names nazwy pakietów powinny być w całości małymi literami, nie wiem na ile ortodoksyjnie w tym projekcie chemy się trzymać takich rzeczy więc piszę to informacyjnie :)
W ogóle polecam plugin do IDE https://plugins.jetbrains.com/plugin/7973-sonarlint , od razu podkreśla i sugeruje co poprawić żeby było zgodnie z oficjalnym codestyle do Javy

Reply:
Oooo! Już sprawdzam plugin! :) A przyznam się, że nie wiedziałam o tym i nie bardzo chyba zwracałam uwagę. Spojrzałam w projekt i z tego, co widzę to jest jedyna taka paczka, więc zaraz poprawię :)